mantidimaging/gui/windows/recon/presenter.py

```diff
@@ -126,7 +126,12 @@ def set_preview_projection_idx(self, idx):
     def set_preview_slice_idx(self, idx):
         self.model.preview_slice_idx = idx
         self.do_update_projection()
-        self.do_preview_reconstruct_slice()
+        # Astra raises type exception during auto fit here, attempting to fit without data to fit to is a problem, it is
+        # not easy to force the method to not be called given no table data.
+        try:
+            self.do_preview_reconstruct_slice()
+        except Exception:
+            pass
```

Comment:
Actually I think it's a better idea to handle this inside `do_preview_reconstruct_slice` itself - and show an error message to the user to check their COR/Tilt values.

Otherwise this exception can be seen raised from the `notify` itself with the steps from the main comments above

Reply:
Ok sounds like a plan!